server/bundles/io.cloudbeaver.service.security/db/cb_schema_update_5.sql

```diff
@@ -10,4 +10,4 @@ CREATE TABLE {table_prefix}CB_AUTH_TOKEN
     PRIMARY KEY (TOKEN_ID),
     FOREIGN KEY (SESSION_ID) REFERENCES {table_prefix}CB_SESSION (SESSION_ID) ON DELETE CASCADE,
     FOREIGN KEY (USER_ID) REFERENCES {table_prefix}CB_USER (USER_ID) ON DELETE CASCADE
-);
+);
```

Comment:
We have schema update 23 in another PR (https://github.com/dbeaver/pro/issues/4118)